sdk/core/Azure.Core/src/Shared/DiagnosticScopeFactory.cs

```diff
@@ -20,18 +20,20 @@ internal class DiagnosticScopeFactory
         private readonly string? _resourceProviderNamespace;
         private readonly DiagnosticListener? _source;
         private readonly bool _suppressNestedClientActivities;
+        private readonly bool _isStable;
 
 #if NETCOREAPP2_1
         private static readonly ConcurrentDictionary<string, object?> ActivitySources = new();
 #else
         private static readonly ConcurrentDictionary<string, ActivitySource?> ActivitySources = new();
 #endif
 
-        public DiagnosticScopeFactory(string clientNamespace, string? resourceProviderNamespace, bool isActivityEnabled, bool suppressNestedClientActivities = true)
+        public DiagnosticScopeFactory(string clientNamespace, string? resourceProviderNamespace, bool isActivityEnabled, bool suppressNestedClientActivities, bool isStable)
```

Comment:
Curious - when would false be passed to `isStable`?

Reply:
Messaging remains experimental and passes `false` here

Reply:
nit: might be helpful to either add a comment or rename parameter to "is experimental" but not a big deal at all

Reply:
sure, added a comment